src/openforms/authentication/contrib/digid_oidc/forms.py

```diff
@@ -0,0 +1,47 @@
+from copy import deepcopy
+
+from django.core.validators import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+from mozilla_django_oidc_db.constants import OIDC_MAPPING as _OIDC_MAPPING
+from mozilla_django_oidc_db.forms import OpenIDConnectConfigForm
+
+from openforms.forms.models import Form
+
+from .models import OpenIDConnectPublicConfig
+
+OIDC_MAPPING = deepcopy(_OIDC_MAPPING)
+
+OIDC_MAPPING["oidc_op_logout_endpoint"] = "end_session_endpoint"
+
+
+class OpenIDConnectPublicConfigForm(OpenIDConnectConfigForm):
```

Comment:
candidate to move to top-level package and maybe even see how much can be merged with the eherkenning variant in a base class